### PR TITLE
Extended the forest storyline with two new options

### PR DESCRIPTION
**Pull request by wlkrt0**
Please review this, @casey-collab

### CODE REVIEW

File: english/forest/fire/woods/woods.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ they are welcome to join the tribe of ancient warlocks.
 
 Maybe you decided to follow the shawman deeper into the forest?
 
+[Follow the shawman](follow/follow.md)
+[Go back to camp](camp/camp.md)
```
Comment (casey-collab) [inline english/forest/fire/woods/woods.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by wlkrt0** [comment]
Thanks for catching my mistake, @casey-collab. I've added a blank line between the new options as requested. Can you please review this again?

**Comment by casey-collab** [comment]
Looks great! :hibiscus:

**Pull request merged**